gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenViewModel.java

```diff
@@ -210,6 +219,11 @@ public void onAddFeatureBtnClick(Point location) {
     addFeatureDialogRequests.setValue(Event.create(location));
   }
 
+  public void onAddPolygonBtnClick(Point location) {
```

Comment:
As per Google (and typical) Java style, please avoid abbreviations for consistency and clarity (`Btn`->`Button`, or better, just remove `Btn`).